nuxt.config.js

```diff
@@ -67,7 +67,26 @@ export default {
 
   tailwindcss: {
     config: {
-      plugins: [tailwindTypography]
+      plugins: [tailwindTypography],
+      theme: {
+        typography: {
+          DEFAULT: { // edit the prose class
```

Comment:
could it be that you need to insert an "extend" keyword here? See https://tailwindcss.com/docs/typography-plugin#customizing-the-css

Reply:
That was it! Thanks :)